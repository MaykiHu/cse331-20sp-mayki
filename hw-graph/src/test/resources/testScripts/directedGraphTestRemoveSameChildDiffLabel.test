#Create a graph
CreateGraph graph1

#Create a node
CreateNode 1

#Create a node
CreateNode 2

#Add a pair of nodes to the graph
AddNode graph1 n1
AddNode graph1 n2

#Add a pair of edges with the same parent-child to the graph, diff labels
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2

#Removes a pair of edges with the same parent-child from the graph, diff labels
RemoveEdge graph1 n1 n2 e1
RemoveEdge graph1 n1 n2 e2