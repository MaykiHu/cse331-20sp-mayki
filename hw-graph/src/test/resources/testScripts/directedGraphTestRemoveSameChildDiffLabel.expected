#Create a graph
created graph graph1

#Add a pair of nodes to the graph
added node n1 to graph1
added node n2 to graph1

#Add a pair of edges with the same parent-child to the graph, diff labels
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1

#Removes a pair of edges with the same parent-child from the graph, diff labels
removed edge e1 from n1 to n2 in graph1
removed edge e2 from n1 to n2 in graph1